onnxruntime/hosting/server_configuration.h

```diff
@@ -84,9 +91,14 @@ class ServerConfiguration {
         << desc << std::endl;
   }
 
+  inline bool file_exists(const std::string& fileName) {
+    std::ifstream infile(fileName.c_str());
+    return infile.good();
```

Comment:
Close the file before return?

Reply:
It's never opened so closing isn't needed.